portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.macro

```diff
@@ -0,0 +1,49 @@
+definition {
+
+	macro changeScreenSize {
```

Comment:
Hey @crislucas1, I think you could modify the name of this macro. For the macro that has a `change` term in the name, it seems to me that the macro is already going to **edit** some information on the page and what you want here is that it just **"Going to simulation > clicking custom"**. So I think you could put another name like `openCustomSimulation` or `goToCustomSimulation` or another idea more following verbs that are **navigation**